src/Subsystems/Lights.cpp

```diff
@@ -26,6 +26,9 @@ Lights::Lights() : Subsystem("Lights") {
     leftUnderGlow = RobotMap::lightsLeftUnderGlow;
     rightUnderGlow = RobotMap::lightsRightUnderGlow;
 
+    heimdallGlow->Set(Relay::kForward);
+    leftUnderGlow->Set(Relay::kForward);
+    rightUnderGlow->Set(Relay::kForward);
```

Comment:
Should the `Lights` subsystem declare it's own type for `Set`? Or we should use `Relay::kOn` instead of `kFroward.